.circleci/config.yml

```diff
@@ -2766,6 +2754,7 @@ workflows:
             branches:
               only:
                 - master
+                - /tagged-release\/.*/
```

Comment:
could we also add a branch prefix like `run-e2e/*` that will automatically run everything __except__ the release? This will prevent us from putting various feature branching in this config in the future and cleaning them up later (as you've done here)

Reply:
One thing I am not 100% sure is if we have lot branches with run-e2e, we would hit the resource limits quickly if they run in parallel

Reply:
Exactly my idea as release != full e2e, but occasionally we could use this, I was thinking of a special commit message to trigger e2e, but branch prefix sounds good and generic as well, the only thing is that whenever a merge happens to a `run-e2e` branch, that must be coordinated with the oncall/release schedule OR we could use a different account

Reply:
Updated the PR to support a tagged release without E2E tests. Now creating a branch with name `tagged-release-without-e2e-tests/some-awesome-feature` should do a NPM release with tag `some-awesome-feature` with out running any E2E tests

Reply:
> could we also add a branch prefix like `run-e2e/*` that will automatically run everything **except** the release? This will prevent us from putting various feature branching in this config in the future and cleaning them up later (as you've done here)

Done!